inspirehep/modules/disambiguation/receivers.py

```diff
@@ -92,7 +92,7 @@ def _needs_beard_reprocessing(authors_before, authors_after):
         return True
 
 
-@after_record_insert.connect
+# @after_record_insert.connect
```

Comment:
Crude solution to a real problem: this receiver fires when a record is inserted, and then the next receiver fires after some other receiver on the same record indexing is run.

Since the disambiguation module is broken anyway, we should not be firing these messy signals.